Airports/EGGD/Ownership.txt

```diff
@@ -1,3 +1,9 @@
+SECTOR:Only GDGND 1:0:0
+OWNER:GDA:GDR:GDT:GDG
```

Comment:
Hi Trev,

Compare this with e.g. line 8 - the "lowest" position should start on the left there and get higher.